DNN Platform/Library/Entities/Profile/ProfileController.cs

```diff
@@ -572,6 +572,9 @@ public static UserInfo UpdateUserProfile(UserInfo user, ProfilePropertyDefinitio
 
                 }
 
+                //Remove the UserInfo from the Cache, as it has been modified
+                DataCache.ClearUserCache(user.PortalID, user.Username);
```

Comment:
The cache clear is already done inside UserUpdate method.  Please check UserController.cs:1993